src/App.tsx

```diff
@@ -1,10 +1,31 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [keyPressed, setKeyPressed] = useState('');
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+  const detectKeyDown = (e: KeyboardEvent) => {
+    setKeyPressed(e.key);
+  };
+
+  useEffect(() => {
+    document.addEventListener('keydown', detectKeyDown);
+  }, []);
```

Comment:
- would be reasonable to cleanup eventListener before component unmounts
- also i'd prefer `keyup` over `keydown` - to not spam events when pressing & holding a button -- keyup only triggers when button is pressed and released.
```suggestion
    document.addEventListener('keydown', detectKeyDown);
    
    return () => document.removeEventListener('keyup', detectKeyDown);
  }, []);
```